src/components/TaskCard/index.jsx

```diff
@@ -0,0 +1,21 @@
+import PropTypes from "prop-types";
+import { formatDate } from "@utils/helpers/formatDate";
+import "./index.scss"
+
+const TaskCard = ({ task }) => {
+  return (
```

Comment:
```suggestion
const TaskCard = ({ task }) => {
 const { title, createdDate } = task

  return (
```

[reference](https://eslint.org/docs/latest/rules/prefer-destructuring)